journal/modernized/01-preface.adoc

```diff
@@ -1,4 +1,5 @@
-== Preface.By John Stoddard
+== Preface. By John Stoddard
+//^^ Does his name need to be here since it's signed below? I also noticed it's omitted in the original edition.
 
 Friendly Reader,
```

Comment:
Here's one thing you missed -- this should be marked as a `[.salutation]` and also in the other edition. 👍